### PR TITLE
Allow docs to access and display code samples

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -4,19 +4,34 @@
...
+This is a boilerplate client application that connects to your Hazelcast Viridian cluster.
+See: https://docs.hazelcast.com/cloud/get-started
 
-See: https://docs.hazelcast.cloud/docs/python-client
+Snippets of this code are included as examples in our documentation,
+using the tag:: comments.
 """
 
 logging.basicConfig(level=logging.INFO)
+# tag::env[]
```
Comment (mdumandag) [inline client.py:15]: So, will the user see this comment when they download the code samples from the cloud?

File: client.py [hunk 2]
```diff
@@ -4,19 +4,34 @@
...
+# Define which environment to use such as production, uat, or dev
```
Comment (mdumandag) [inline client.py:16]: This property is only set for internal testing, it shouldn't be visible to the user. Is this the case?

File: client.py [hunk 3]
```diff
@@ -4,19 +4,34 @@
...
 client = hazelcast.HazelcastClient(
     cluster_name="YOUR_CLUSTER_NAME",
+    # The cluster discovery token is a unique token
+    # that maps to the current IP address of the cluster.
+	# Cluster IP addresses may change.
+	# This token allows clients to find out the current IP address
+	# of the cluster and connect to it.
     cloud_discovery_token="YOUR_CLUSTER_DISCOVERY_TOKEN",
+    # Allow the client to collect metrics
+	# so that you can see client statistics in Management Center.
```
Comment (mdumandag) [inline client.py:30]: Why is this (and some others) are indented with an extra tab?

Comment (JakeSCahill) [reply client.py:30]: will fix that

File: client.py (hunk 2 above)
Comment (JakeSCahill) [reply client.py:16]: hmm I see. So this line gets stripped out before it's uploaded to production clusters?

File: client.py (hunk 1 above)
Comment (JakeSCahill) [reply client.py:15]: I added this so we can filter out that line. But if our release process already does that, then I think we can remove it.

**Comment by JakeSCahill** [comment]
I see that we don't use `master`. Is it better if I resolve your comments and submit a new PR against 5.1?

**Comment by JakeSCahill** [comment]
Closing this for now as we will need dev involvement to remove our tags.

**Pull request closed without merging**